Builds/containers/ubuntu-builder/ubuntu_setup.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+set -ex
+
+source /etc/os-release
+
+if [[ ${VERSION_ID} =~ ^18\. || ${VERSION_ID} =~ ^16\. ]] ; then
+    echo "setup for ${PRETTY_NAME}"
+else
+    echo "${VERSION} not supported"
+    exit 1
+fi
+
+export DEBIAN_FRONTEND="noninteractive"
+echo "Acquire::Retries 3;" > /etc/apt/apt.conf.d/80-retries
+echo "Acquire::http::Pipeline-Depth 0;" >> /etc/apt/apt.conf.d/80-retries
+echo "Acquire::http::No-Cache true;" >> /etc/apt/apt.conf.d/80-retries
+echo "Acquire::BrokenProxy    true;" >> /etc/apt/apt.conf.d/80-retries
+apt-get update -o Acquire::CompressionTypes::Order::=gz
+
+apt-get -y update
+apt-get -y install apt-utils
+apt-get -y install software-properties-common wget
+apt-get -y upgrade
+if [[ ${VERSION_ID} =~ ^18\. ]] ; then
+    apt-add-repository -y multiverse
+    apt-add-repository -y universe
+elif [[ ${VERSION_ID} =~ ^16\. ]] ; then
+    add-apt-repository -y ppa:ubuntu-toolchain-r/test
+fi
+apt-get -y clean
+apt-get -y update
+
+apt-get -y --fix-missing install  \
+    make cmake ninja-build ccache \
+    protobuf-compiler libprotobuf-dev libssl-dev libzstd-dev \
+    python-pip \
+    gdb gdbserver \
+    libstdc++6 \
+    flex bison graphviz graphviz-dev \
+    xauth libgl1-mesa-dev mesa-common-dev xvfb \
+    libicu-dev texinfo \
+    java-common javacc \
+    gcc-7 g++-7 \
+    gcc-8 g++-8 \
+    dpkg-dev debhelper devscripts fakeroot \
+    debmake git-buildpackage dh-make gitpkg debsums gnupg \
+    dh-buildinfo dh-make dh-systemd
```

Comment:
Just curious, why do we need `graphviz`, `mesa` and `java`?

Reply:
removed (leftover requirement for CI that needed qt libs)